packages/bff/src/services/authorizationService.ts

```diff
@@ -0,0 +1,37 @@
+import { authorizationApi } from "pagopa-interop-api-clients";
+import { AuthorizationProcessClient } from "../providers/clientProvider.js";
+
+export const getAllClients = async (
+  authorizationProcessClient: AuthorizationProcessClient,
+  consumerId: string,
+  purposeId: string,
+  headers: { "X-Correlation-Id": string }
+): Promise<authorizationApi.ClientWithKeys[]> => {
+  const getClientsFrom = async (
```

Comment:
Also all this recursive get from paginated endpoints is something we do often in BFF/API GW - here I created a utility, we could do the same for BFF or put it in commons: https://github.com/pagopa/interop-be-monorepo/pull/764/files#diff-11af88c890bfbedf173417f6460899659acfb3ec95ba02ca4db188340390ea88R3